.github/workflows/crux-mir-build.yml

```diff
@@ -136,13 +136,22 @@ jobs:
         run: .github/ci.sh test crux-mir
 
       - shell: bash
-        run: .github/ci.sh bundle_crux_mir_files
+        run: |
+          NAME="crux-mir-${{ needs.outputs.outputs.crux-mir-version }}-${{ matrix.os }}-x86_64"
+          echo "NAME=$NAME" >> $GITHUB_ENV
```

Comment:
It's unlikely to happen here, but in the spirit of full safety I've learned that `declare -p NAME | cut -d' ' -f2-` is usually better than `echo "NAME=$NAME"` because the former will escape things like double-quotes in the value of NAME.

Same comment for the other yaml file.

Reply:
`f3` seems to do what I want, so I've opted for that instead.

Reply:
Hm, that's [still not quite right](https://github.com/GaloisInc/crucible/runs/8115069198?check_suite_focus=true#step:22:23):

```
+ sign '"crux-llvm-0.6.0.99-ubuntu-20.04-x86_64".tar.gz'
+ set +x
gpg: inserting ownertrust of 6
gpg: using "07FC90D0C93F5F3B56354E1DCBF4BB4143DB4A3A" as default secret key for signing
gpg: can't open '"crux-llvm-0.6.0.99-ubuntu-20.04-x86_64".tar.gz': No such file or directory
gpg: signing failed: No such file or directory
```